WoWsShipBuilder.ViewModels/ShipVm/UpgradePanelViewModelBase.cs

```diff
@@ -20,7 +20,7 @@ public UpgradePanelViewModelBase(Ship ship, Dictionary<string, Modernization> up
             List<Modernization> filteredModernizations = upgradeData.Select(entry => entry.Value)
                 .Where(m => !(m.BlacklistedShips?.Contains(ship.Name) ?? false))
                 .Where(m => m.ShipLevel?.Contains(ship.Tier) ?? false)
-                .Where(m => m.AllowedNations?.Contains(ship.ShipNation) ?? false)
+                .Where(m => ship.ShipNation == Nation.Common || (m.AllowedNations?.Contains(ship.ShipNation) ?? false))
```

Comment:
Can a ship ever have Nation.Common?

Reply:
Don't think so. Probably only special AI driven ships like operations or events ones may have it